### PR TITLE
Add a new category for chairs in the GOVERNANCE

### PR DESCRIPTION
**Pull request by kmova**
In an complex turn of events, most of the maintainers had to move away (due to various reasons) from actively being  involved in the project in day-to-day basis. The maintainers were still involved in the background/slack channel providing the needed help for the contributors to step up as maintainers. This involvement was hard to capture in github stats.

The lack of github involvement from listed maintainers and coupled with missing of a proper transition on community aspects to the new contributors resulted in a "justified" perception that project isn't active. 

To avoid this from happening again, one way is to clearly communicate the level of involvement of active maintainers (existing definition) vs those providing mentorship/guidance.

I am proposing to: 
- Include a category for chairs that are still associated with the project in terms of shepherding the direction/governance of the project.

Audit the governance at a regular intervals to:
- Move the maintainers that have to step down for a while into chairs. 
- Move the maintainers that can't be involved in anyway with the project going forward to Emeritus status 
- Add the contributors that are stepping up as maintainers.

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -72,6 +72,19 @@ depend on and trust to make decisions in the best interest of the project.
 
 Maintainers have admin access to the GitHub organization.
 
+### Chairs
+
+OpenEBS Chairs are ex-maintainers that have stepped down from active contributions,
```
Comment (avishnu) [inline GOVERNANCE.md:77]: ex-maintainers and decision-makers

Comment (kmova) [reply GOVERNANCE.md:77]: maintainers are responsible for decisions. Probably we don't need to call it out?

Comment (avishnu) [reply GOVERNANCE.md:77]: The wording says "ex-maintainers", hence the comment.

**Review by avishnu — APPROVED** [review]
lgtm

**Review by vishnuitta — APPROVED** [review]
Changes are good. Thanks Kiran.

**Pull request merged**